src/styles/blocks/product_card.css

```
@@ -0,0 +1,93 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__link-img {
+  display: flex;
+  justify-content: center;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+
+}
+
+.product-card__title {
+  font-size: inherit;
+  margin: 0;
+}
+
+.product-card__title-link {
+  display: inline-block;
+  text-decoration: none;
+  font-weight: 500;
+  line-height: calc(18 / 12 * 100%);
```

Можна і без відсотків
```suggestion
  line-height: calc(18 / 12 );
```

---

добре, дякую)